tests/e2e/README.md

```diff
@@ -17,20 +17,33 @@ Run `make test-e2e` to execute the tests.
 | `project create` | Y |
 | `project get` | Y |
 | `project delete` | Y |
+| `project list` | N |
+| `project set` | N |
 | `app create`  | Y  |
 | `app get`  | Y  |
 | `app set` | Y |
 | `app delete` | Y |
+| `app describe` | N |
 | `app list` | Y |
-| `catalog` | N |
-| `create --git` | N |
+| `catalog list` | Y |
+| `catalog search` | N |
+| `component get` | Y |
+| `component set` | Y |
+| `create --git` | Y |
 | `create --local` | Y |
+| `create --binary` | Y |
+| `delete` | N |
+| `describe` | N |
+| `link` | Y |
 | `list` | Y |
-| `push` | Y |
-| `storage add` | N |
-| `storage list` | N |
-| `storage remove` | N |
-| `update` | N |
+| `push` --local | Y |
+| `push` --git | Y |
+| `push` --binary | Y |
+| `storage create` | Y |
+| `storage list` | Y |
+| `storage delete` | N |
+| `update` | Y |
```

Comment:
we need to track tests for all directions.
We have try types of component source and we need to tests all posible update permutations.


from `--git` to `--local`
from `--local` to `--git`
from `--binary` to `--git`
from `--git` to `--binary`
from `--local` to `--binary`
from `--binary` to `--local`

Reply:
Sure, will add them